src/skibidi_orm/migration_engine/state_manager/state_manager.py

```diff
@@ -20,6 +20,13 @@
 
 
 class StateManager[TTable: BaseTable[BaseColumn[Any]]](IStateManager):
+    """
+    Class that will analyze the diffrences between ta database schema and a class structure defined schema
+    when instantiated.
+
+    It can provide a list of operations needed to transform the first database into the second one
```

Comment:
maybe to transform the schema of the database to be equivalent with the one defined via python objects and migration files?